### ISSUE TITLE
__test__/web/css/styles.css crashes the static-content:deploy

### ISSUE THREAD

**Issue by aronesztergomi** [SnowdogApps/magento2-theme-blank-sass#111]
Hi There,

From that commit:
https://github.com/SnowdogApps/magento2-theme-blank-sass/commit/436a36dfc1904c313c6b0bca44995b6d167f068e

I got this exception:

`[UnexpectedValueException]                                                                                                                  
 Could not parse theme static file '.../vendor/snowdog/theme-blank-sass/__test__/web/css/styles.css'`

If I remove `__test__` dir, everything works perfectly.

**Comment by Igloczek** [comment]
Are you sure that your theme is compiled well and doesn't contain any errors in this `styles.css` file?
I'm not sure if there is a way, to exclude some CSS from processing in `static-content:deploy`, previously this files was just saved in same directory as `*.scss` files and was processed well.

**Comment by danslo** [comment]
Can confirm this error. Just running something like:

``` bash
php -f bin/magento -- setup:static-content:deploy nl_NL en_US -t Magento/backend -t MyTheme/default
```

**Comment by Igloczek** [comment]
It looks like `setup:static-content:deploy` doesn't like when directory have `_` in name... 😞 
Simple rename `__test__` to `test` or `.test` fixes problem.

**Comment by Igloczek** [comment]
Fixed in 0.9.2

**Comment by aronesztergomi** [comment]
Thanks!
